tests/runtests.sh

```diff
@@ -149,7 +154,17 @@ function start_selenium() {
     done
 }
 
-tempdir=$(mktemp -d --suffix toastertest --tmpdir)
+if [ "" != "$LOCALDIR" ]; then
```

Comment:
This whole section down to line 167 is pretty ugly. Is there a way we can clean it up?

And is there a reason we can't keep the "toasterbuild" directory even in local mode? Is it because you're trying to reuse a build directory that may have already been created?